python/action-server/test_trigger.py

```diff
@@ -0,0 +1,22 @@
+import json
+import yaml
+from state_machine import StateMachine
+import main
+
+def test_statechart():
+    with open('action.yml') as file:
+        try:
+            settingConfig = yaml.safe_load(file)  
+        except yaml.YAMLError as err:
+            print(err)
+
+    settings = main.ActionSettings(**settingConfig)
+    session = main.Session(settings)
+    session.setup_action_server()
+    model = StateMachine()
+    model.init_session(session)
+    value = session.get('/trigger/start')
```

Comment:
Also endpoints are going to be of the form `BASE_KEY_EXPR/trigger`, not `/trigger`, where BASE_KEY_EXPR should come from the config file and be of the form `Genotyper/id/ServerName/id/` example: `Genotyper/0/DNASensor/`’